lib/helper/Puppeteer.js

```diff
@@ -1985,6 +1985,31 @@ class Puppeteer extends Helper {
   async grabDataFromPerformanceTiming() {
     return perfTiming;
   }
+
+  /**
+   * {{> grabElementSize }}
+   *
+   */
+  async grabElementSize(locator, prop) {
+    let els = await this._locate(locator);
+    els = await Promise.all(els.map(el => el.boundingBox()));
+    const result = [];
+    els = els.filter(v => v);
+
+    if (prop) {
+      els.forEach((element) => {
+        result.push(element[prop]);
+      });
+    } else {
+      els.forEach((element) => {
+        delete element.x;
```

Comment:
Oh no) 
x and y can be use! My command used this property)) 
I thought you make x and y in webdriverio)))

Reply:
X and y coordinates are important))

Reply:
haha. I'm thinking of renaming this method as `grabElementInfo` to convey better meaning. Cause we want to return the element size and location(x,y). Do you have any suggestions?

Reply:
Maybe grabElementBoundingRect ?

Reply:
sounds nice. I give a go for it.

Reply:
@Vorobeyko done. not both wdio and puppeteer return location, weight and height. :)